core/src/main/scala/org/bitcoins/core/psbt/PSBT.scala

```diff
@@ -1436,7 +1449,25 @@ case class InputPSBTMap(elements: Vector[InputPSBTRecord]) extends PSBTMap {
       signers: Vector[Sign],
       conditionalPath: ConditionalPath = ConditionalPath.NoConditionsLeft): UTXOSpendingInfoFull = {
     require(!isFinalized, s"Cannot update an InputPSBTMap that is finalized")
+    val spendingInfoSingle =
+      toUTXOSpendingInfoSingle(txIn, signers.head, conditionalPath)
+
+    BitcoinUTXOSpendingInfoFull(
+      outPoint = spendingInfoSingle.outPoint,
+      output = spendingInfoSingle.output,
+      signers = signers,
+      redeemScriptOpt = spendingInfoSingle.redeemScriptOpt,
+      scriptWitnessOpt = spendingInfoSingle.scriptWitnessOpt,
+      hashType = spendingInfoSingle.hashType,
+      conditionalPath = conditionalPath
+    )
```

Comment:
If you want, feel free to add a util method to some companion object that does this. For example you could add another constructor for `BitcoinUTXOSpendingInfoFull` that takes in a `Vector[Sign]` and a `BitcoinUTXOSpendingInfoSingle` and does this.